assembl/static2/flow/object_types.js

```diff
@@ -113,3 +113,13 @@ type Extract = {
   extractAction: string,
   body: string
 };
+
+type ConfigurableField = {
```

Comment:
I think we should create a task about flow types. We should probably create multiple files for types relative to different parts of the code